vscode-ext/vscode-ext

```diff
@@ -6,5 +6,7 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+- Extension provides quick select for setting impersonated container name (environment variable `MIRRORD_IMPERSONATED_CONTAINER_NAME`)
```

Comment:
We don't maintain this file - we document all changes in the main mirrord CHANGELOG file.

Reply:
Removed this change. I'll just keep the change in that main CHANGELOG.